benchmark/Cajita_FastFourierTransformPerformance.cpp

```diff
@@ -0,0 +1,221 @@
+/****************************************************************************
+ * Copyright (c) 2018-2021 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "Cabana_BenchmarkUtils.hpp"
+
+#include <Cajita_Array.hpp>
```

Comment:
I would just include `Cajita.hpp`

Reply:
Thanks @streeve for your helpful review
I made changes to address your comments, and can clean up commits if you think the PR is in good shape to merge.
I'll look at the allocation for FFTs and try to add a work buffer in a follow-on PR